src/StdUtils.sol

```diff
@@ -19,6 +19,7 @@ abstract contract StdUtils {
         57896044618658097711785492504343953926634992332820282019728792003956564819968;
     uint256 private constant UINT256_MAX =
         115792089237316195423570985008687907853269984665640564039457584007913129639935;
+    address private constant DEFAULT_CREATE2_DEPLOYER = 0x4e59b44847b379578588920cA78FbF26c0B4956C;
```

Comment:
Nit, maybe we call this `CREATE2_FACTORY` to match the version in `Base.sol`, and we can add a comment explaining this is the default one used by forge?

https://github.com/foundry-rs/forge-std/blob/066ff16c5c03e6f931cd041fd366bc4be1fae82a/src/Base.sol#L30